src/main/scala/shine/DPIA/FunctionalPrimitives/AbstractScan.scala

```diff
@@ -88,7 +88,7 @@ abstract  class AbstractScan(n: Nat,
       </input>
     </reduce>.copy(label = {
       val name = this.getClass.getSimpleName
-      Character.toLowerCase(name.charAt(0)) + name.substring(1)
+      s"${Character.toLowerCase(name.charAt(0))}${name.substring(1)}"
     })
```

Comment:
maybe we can open an issue to remove the XML printing altogether?

Reply:
Yes, obviously orthogonal, but I agree: https://github.com/rise-lang/shine/issues/69